packages/node_modules/glimmer-runtime/lib/syntax/core.ts

```diff
@@ -420,8 +421,13 @@ export class DynamicAttr extends AttributeSyntax {
   }
 
   compile(compiler: CompileInto, env: Environment) {
-    compiler.append(new PutValueOpcode({ expression: this.value.compile(compiler, env) }));
-    compiler.append(new DynamicAttrOpcode(this));
+    let {namespace, value} = this;
+    compiler.append(new PutValueOpcode({ expression: value.compile(compiler, env) }));
+    if (namespace) {
+      compiler.append(new DynamicAttrNSOpcode(this));
+    } else {
+      compiler.append(new DynamicAttrOpcode(this));
+    }
```

Comment:
Good catch! I guess that means we didn't have tests for this case before :fearful: